.circleci/soltest_all.sh

```diff
@@ -70,6 +70,12 @@ do
     for EVM in ${EVM_VALUES[@]}
     do
         [[ " $RUN_STEPS " =~ " $STEP " ]] && EVM="$EVM" OPTIMIZE="$OPTIMIZE" BOOST_TEST_ARGS="-t !@nooptions" "${REPODIR}/.circleci/soltest.sh"
+        EWASM_ARGS=""
+
+        # run tests against hera ewasm evmc vm, only if OPTIMIZE == 0 and evm version is byzantium
```

Comment:
Is the idea that we only run a single combination on ewasm? Then I would recommand to not use `soltest_all.sh` but `soltest.sh` directly.